ui/job-view/pushes/PushList.jsx

```diff
@@ -168,6 +166,7 @@ class PushList extends React.Component {
           ))}
         {loadingPushes && (
           <div
+            aria-label="progress bar"
```

Comment:
This is indeed a progress bar, but since it's used by screen readers I think it'd be more helpful to give context on what the progress bar is for.